src/app/components/client/system-report-modal/system-report-form.vue

```diff
@@ -1,5 +1,24 @@
 <template>
 	<app-form name="systemReportForm">
+		<app-form-group name="issue_tracker_url" optional :label="$gettext('Issue Tracker Link')">
+			<p class="help-block">
+				<translate>
+					Paste the link to your ticket here. It's optional, but without it we can't get back to you
```

Comment:
I think we should directly link to the issue tracker project here.

Reply:
We do in the modal that uses the form so its clear enough. It's also the only link there so it stands out too.